include/libtorrent/file_storage.hpp

```diff
@@ -306,6 +307,7 @@ namespace aux {
 			, file_flags_t file_flags = {}
 			, std::time_t mtime = 0, string_view symlink_path = string_view()
 			, char const* root_hash = nullptr);
+#endif
```

Comment:
you would have to surround the definitions of these functions with this preprocessor condition too, right?

Reply:
otherwise I imagine you would get a compile error

Reply:
that's right!

Reply:
done